### PR TITLE
set up continuous integration

### PR DESCRIPTION
**Pull request by Jonathansteven21**

### CODE REVIEW

File: cypress.config.ts [hunk 1]
```diff
@@ -4,12 +4,11 @@ export default defineConfig({
...
+    // solo necesitas agregar estas dos propiedades, no incluir este comentario
+    video: false,
+    screenshotOnRunFailure: false,
     setupNodeEvents(on, config) {
-      // modify config values examples
-      // config.defaultCommandTimeout = 10000
-
-      // IMPORTANT return the updated config object
-      return config;
+      // Codigo existente no modificar
```
Comment (danielgalvis98) [inline cypress.config.ts]: The comment on the inscruction means that on this section there was a code that should not be change, not to put that comment. Please revert the change of this section.

File: cypress.config.ts [hunk 2]
```diff
@@ -4,12 +4,11 @@ export default defineConfig({
   // setupNodeEvents can be defined in either
   // the e2e or component configuration
   e2e: {
+    // solo necesitas agregar estas dos propiedades, no incluir este comentario
+    video: false,
+    screenshotOnRunFailure: false,
```
Comment (danielgalvis98) [inline cypress.config.ts]:
```suggestion
    video: false,
    screenshotOnRunFailure: false,
```

**Review by holgiosalos — APPROVED** [review]
